config/locales/en.yml

```diff
@@ -84,6 +84,7 @@ en:
     tag:
       dev: Others
       ui: Internal Meeting
```

Comment:
we can't drop this translation

Reply:
I wanted to keep this to avoid translation missing in versioning changes. After update migration that changed `ui` -> `im` in WorkTime enum model, there are still `ui` keys in PaperTrail versions